src/get-started/learn-more.md

```diff
@@ -1,54 +1,163 @@
 ---
-title: Learn more
-description: More resources to help you learn Flutter.
+title: Learn
+description: Resources to help you learn Flutter.
 prev:
   title: Write your first Flutter app
   path: /get-started/codelab
 ---
 
-Learn more about the Flutter framework from the following pages:
-
-## Flutter basics
-
-* [Introduction to Flutter's UI][]
-* [Building layouts][] tutorial
-* [Add interactivity][] tutorial
-
-## Apply your existing knowledge
-
-* [Flutter for Android developers][]
-* [Flutter for SwiftUI developers][]
-* [Flutter for UIKit developers][]
-* [Flutter for React Native developers][]
-* [Flutter for web developers][]
-* [Flutter for Xamarin.Forms developers][]
-
-## Other resources
-
-* [Flutter samples][]
-* [Flutter cookbook][]
-* [Bootstrap into Dart: learn more about the language][]
-* [Flutter API Docs][]
-* [The Complete Flutter Developer Bootcamp Using Dart][]
-* [Udacity online Flutter training][]
-
-Reach out to us at our [mailing list][]. We'd love to hear from you!
-
-Happy Fluttering!
-
-[Add interactivity]: {{site.url}}/ui/interactivity
-[Bootstrap into Dart: learn more about the language]: {{site.url}}/resources/bootstrap-into-dart
-[Building layouts]: {{site.url}}/ui/layout/tutorial
-[The Complete Flutter Developer Bootcamp Using Dart]: https://www.appbrewery.co/p/flutter-development-bootcamp-with-dart
-[Flutter API Docs]: {{site.api}}
-[Flutter cookbook]: {{site.url}}/cookbook
-[Flutter for Android developers]: {{site.url}}/get-started/flutter-for/android-devs
-[Flutter for SwiftUI developers]: {{site.url}}/get-started/flutter-for/swiftui-devs
-[Flutter for UIKit developers]: {{site.url}}/get-started/flutter-for/uikit-devs
-[Flutter for React Native developers]: {{site.url}}/get-started/flutter-for/react-native-devs
-[Flutter samples]: https://flutter.github.io/samples
-[Flutter for web developers]: {{site.url}}/get-started/flutter-for/web-devs
-[Flutter for Xamarin.Forms developers]: {{site.url}}/get-started/flutter-for/xamarin-forms-devs
-[Introduction to Flutter's UI]: {{site.url}}/ui
-[mailing list]: mailto:{{site.email}}
-[Udacity online Flutter training]: https://www.udacity.com/course/build-native-mobile-apps-with-flutter--ud905
+## Layout
```

Comment:
When we start adding other topics, should this be one long page or multiple pages? 

Pros of multiple pages (imo):

- Future proofing, assuming we plan on adding a lot to this over time.
- Easier to navigate for folks who don't want to sit down for 20 hours straight.
- Easier to maintain

Cons of multiple pages:

- We'll end up with a new sub-sidenav that looks very similar to the current Sidenav.

Reply:
Alternatively, we can do something like Android did with expansion panels:
https://developer.android.com/courses/android-basics-compose/course

(I have _serious_ misgivings about the content of that page and I would really like to avoid using it is a guide, but it does have nice UI)

Reply:
Doing even an "MVP" implementation of UI similar to the Android course page is relatively significant amount of work. I highly recommend we come to an agreement on what the page should look like before spending time on this.

Reply:
I have been imagining a "learn more" page with links to individual topic pages. Doesn't mean we have to do it that way, but it could help with analytics about which topics get traffic, and probably make it easier for mobile readers.